### ISSUE TITLE
Making Wiki-Go more accessible to blind users who use screenreaders

### ISSUE THREAD

**Issue by domasofan** [leomoon-studios/wiki-go#159]
Hi all,

Just discovered Wiki-Go and tested a few things.
I quite like it how it works.
I figured out some problems.
Maybe someone could have a look at those things.

* The editor (CodeMirror) seems not to be all that accessible in the wiki. According to the website it should be accessible to screenreaders though. And the demo on the CodeMirror website seems to work well.
* Some unlabeled buttons in/above the eeditor. Though when you move over them by hitting tab they seem to have labels. maybe label it like the new, edit, settings, print, etc buttons in page view.
* Some sporadic unlabeled buttons in the settings and other places
* No checkboxes, but just clickable elements. Maybe they can be replaced by checkboxes or toggle buttons so screenrreaders can detect if they are on or off.

Maybe you have someone in your team to make it more accessible for blind users.
That would be great.

Greetings and thanks in advance,
Simon
